File: app/src/main/java/com/jingxi/smartlife/pad/mvp/home/data/LockADBean.java
package com.jingxi.smartlife.pad.mvp.home.data;

import com.google.gson.annotations.SerializedName;
import com.wujia.businesslib.data.RootResponse;

import java.util.ArrayList;

/**
 * author ：shenbingkai@163.com
 * date ：2019-04-13
 * description ：
 */
public class LockADBean extends RootResponse {

    public AD data;

    public static class AD{
        @SerializedName("cover")
        public String image;
        public String url;

//        public int imageType;
    }
}

File: app/src/main/java/com/jingxi/smartlife/pad/mvp/home/data/LockADBean.kt
package com.jingxi.smartlife.pad.mvp.home.data

import com.google.gson.annotations.SerializedName
import com.wujia.lib_common.base.RootResponse

/**
 * author ：shenbingkai@163.com
 * date ：2019-04-13
 * description ：
 */
class LockADBean : RootResponse() {

    var data: AD? = null

    class AD {
        @SerializedName("cover")
        var image: String? = null
        var url: String? = null

        //        public int imageType;
    }
}
